dataproc_bootstrap/fx_usage_init.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -exo pipefail
+
+# Logs will be available on the dataproc nodes at /var/log/dataproc-initialization-script-X.log
+# or via the GCP Dataproc UI
+
+/opt/conda/default/bin/pip install --upgrade pip
+
+/opt/conda/default/bin/pip install arrow==0.10.0
```

Comment:
Can we use the requirements files from the repo instead? Then we're guaranteed to have the same dependencies between dev and prod.

Reply:
The repo's dockerfile and the actual libraries on the emr hosts that the job runs on are different. I went with the later

Reply:
also there were some changes when test iterating on dataproc. So the init script here is version i got to work

Reply:
We should go with the former (and it looks like most of these are the former?). It makes it easier when testing and making changes to match dev.

Reply:
I disagree. We should go with the actual working instance on prod. Plus the dev container is way faster to iterate on than actually running the workload on dataproc